gradio/utils.py

```diff
@@ -1002,7 +1016,7 @@ def get_markdown_parser() -> MarkdownIt:
                 "linkify": True,
                 "typographer": True,
                 "html": True,
-                "breaks": True,
```

Comment:
Or 

```py
import gradio as gr

response = """
Hello, here's an example of a program in python:
\n\n
And here's the reponse
"""

with gr.Blocks() as demo:
    chatbot = gr.Chatbot([
        ("abc", response)
    ])
demo.launch()
```

Reply:
Actually thinking about this some more, I actually think that although this is a breaking change, it actually improves the behavior of Markdown and the Chatbot because it prevents extra newlines from being displayed. I think you almost never want to have multiple new lines, but if you do, you can get that by manually putting `<br>`

Reply:
So imo let's go with this! But let's put a comment in the breaking changes section of the changelog

Reply:
`"breaks": True` converts `\n` to <br> after processing the message. Is it really necessary?